webapp/greenhouse_api.py

```diff
@@ -51,6 +55,34 @@ def get_vacancy(job_id):
     return job
 
 
+def submit_to_greenhouse(form_data, form_cv, job_id="1383152"):
+    # Encode the API_KEY to base64
+    auth = (
+        "Basic " + str(base64.b64encode(API_KEY.encode("utf-8")), "utf-8")[:-2]
```

Comment:
Currently, the error I see if I don't provide the API key is:

```
AttributeError: 'NoneType' object has no attribute 'encode'
```

If you simply make this `os.environ["GREENHOUSE_API_KEY"]` then we'd get a nicer error like:

```
KeyError: 'GREENHOUSE_API_KEY'
```